charts/yandex-csi-controller/README.md

```diff
@@ -0,0 +1,52 @@
+# cloud provisioning
+
+```bash
+export YANDEX_SA_DATA=$(yc iam service-account create --name k8s-csi-controller --format json )
+export YANDEX_SA_ID=$(echo $YANDEX_SA_DATA | jq .id | sed 's\"\\g')
+export YANDEX_SA_FOLDER=$(echo $YANDEX_SA_DATA | jq .folder_id | sed 's\"\\g')
+
+
+yc resource-manager folder add-access-binding ${YANDEX_SA_FOLDER} \
+  --role admin \
+  --subject serviceAccount:${YANDEX_SA_ID}
+
+yc iam key create --service-account-name k8s-csi-controller  --output k8s-csi-controller-key.json
+
+export serviceAccountJSON=$(cat k8s-csi-controller-key.json)
+```
+
+# Install
+
+Read the k8s-csi-controller-key.json file and load the content instead of the example in the block <service-account-json>
+
+```bash
+
+export NAMESPACE=kube-system
+
+cat <<EOF | kubectl apply -f -
```

Comment:
Can't we just pass this as a `--set` to Helm chart?

Reply:
fix it